client/i18n/locales/english/intro.json

```
@@ -344,6 +344,9 @@
         "title": "Learn Data Structures By Building A Shopping Cart",
         "intro": ["", ""]
       },
+      "learn-fetch-and-promises-by-building-an-fcc-authors-page": {
+        "title": "Learn Fetch and Promises By Building an fCC Authors Page"
+      },
```

```suggestion
      "learn-fetch-and-promises-by-building-an-fcc-authors-page": {
        "title": "Learn Fetch and Promises By Building an fCC Authors Page",
        "intro": ["", ""]
      },
```

It lost the intro array which prevents the block from loading correctly.